### PR TITLE
new package for Ookla Speedtest CLI 1.2.0

### PR DESCRIPTION
**Pull request by KhoobBabe**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96346)

### CODE REVIEW

File: manifests/o/Ookla/Speedtest/CLI/1.2.0/Ookla.Speedtest.CLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Ookla.Speedtest.CLI
+PackageVersion: 1.2.0
```
Comment (mdanish-kh) [inline manifests/o/Ookla/Speedtest/CLI/1.2.0/Ookla.Speedtest.CLI.installer.yaml:5]:
```suggestion
PackageVersion: 1.2.0
AppsAndFeaturesEntries:
  - DisplayVersion: 1.2.0.84
```
The package is marketed as `1.2.0` but `speedtest --version` shows `1.2.0.84`, so we'll use AppsAndFeaturesEntries here

File: manifests/o/Ookla/Speedtest/CLI/1.2.0/Ookla.Speedtest.CLI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+PublisherUrl: https://www.speedtest.net
+PublisherSupportUrl: https://www.speedtest.net/help
+PrivacyUrl: https://www.speedtest.net/about/privacy
+Author: Ookla
+PackageUrl: https://www.speedtest.net/apps/cli
+LicenseUrl: https://www.speedtest.net/about/terms
+Copyright: Copyright (c) Ookla, LLC.
+CopyrightUrl: https://www.speedtest.net/about/terms
+PackageName: Speedtest CLI
+License: proprietary
```
Comment (mdanish-kh) [inline manifests/o/Ookla/Speedtest/CLI/1.2.0/Ookla.Speedtest.CLI.locale.en-US.yaml:17]:
```suggestion
License: Proprietary
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Comment by stephengillie** [comment]
The package requires the user to accept a license on first launch. It's reminiscent of Sysinternals, but with a CLI prompt instead of a GUI. 
```
==============================================================================

You may only use this Speedtest software and information generated
from it for personal, non-commercial use, through a command line
interface on a personal computer. Your use of this software is subject
to the End User License Agreement, Terms of Use and Privacy Policy at
these URLs:

        https://www.speedtest.net/about/eula
        https://www.speedtest.net/about/terms
        https://www.speedtest.net/about/privacy

==============================================================================

Do you accept the license? [type YES to accept]:
```

The installer doesn't seem to have a flag for skipping this prompt.

**Pull request merged**